examples/magento-graphcms/pages/[...url].tsx

```diff
@@ -68,6 +54,8 @@ function CategoryPage(props: CategoryProps) {
   const isLanding = category?.display_mode === 'PAGE'
   const page = pages?.[0]
   const isCategory = params && category && products?.items && filterTypes
+  const theme = useTheme()
+  const matches = useMediaQuery(theme.breakpoints.up('md'))
```

Comment:
`useMediaQuery` is not allowed to be used to determine if elements should be shown on the frontend. Use CSS to determine if things should be shown.

useMediaQuery uses Javascript to determine this and thus will create a Layout shift when loading either on mobile or desktop.